src/transaction/client_credentials.hpp

```diff
@@ -48,7 +48,9 @@ enum boot_client_type
   BOOT_CLIENT_RO_BROKER_REPLICA_ONLY = 12,
   BOOT_CLIENT_SO_BROKER_REPLICA_ONLY = 13,
   BOOT_CLIENT_ADMIN_CSQL_WOS = 14,	/* admin csql that can write on standby */
-  BOOT_CLIENT_DDL_PROXY = 15
+  BOOT_CLIENT_DDL_PROXY = 15,
+  BOOT_PSEUDO_CLIENT_REPL_APPLIER = 16,
```

Comment:
Do we need the additional client types?

Reply:
I will keep only BOOT_PSEUDO_CLIENT_REPL_COPIER
I opened http://jira.cubrid.org/browse/CBRD-23214